src/Microsoft.Azure.SignalR/ServiceOptions.cs

```diff
@@ -120,5 +120,11 @@ public int ConnectionCount
         /// Gets or sets a function which accepts <see cref="HttpContext"/> and returns a bitmask combining one or more <see cref="HttpTransportType"/> values that specify what transports the service should use to receive HTTP requests.
         /// </summary>
         public Func<HttpContext, HttpTransportType> TransportTypeDetector { get; set; } = null;
+
+        /// <summary>
+        /// Allow clients enable stateful reconnects for connecting to service.
```

Comment:
Add more comments about how this works with the aspnetcore's configure https://learn.microsoft.com/en-us/aspnet/core/signalr/configuration?view=aspnetcore-8.0&tabs=dotnet#configure-stateful-reconnect?

Also there is one option in aspentcore `StatefulReconnectBufferSize`, do we want to support it?

In aspnetcore, the configures could be set per hub, is it also supported?

Reply:
we do not support customize `StatefulReconnectBufferSize` for now.
we may add it in later version.